packages/aws-library/src/aws_library/ec2/models.py

```diff
@@ -13,13 +13,12 @@
     Extra,
     Field,
     NonNegativeFloat,
-    PositiveInt,
     validator,
 )
 from types_aiobotocore_ec2.literals import InstanceStateNameType, InstanceTypeType
 
 
-class Resources(BaseModel):
+class Resources(BaseModel, frozen=True):
```

Comment:
there is actually no reason, it is because I got confused between dataclasses and pydantic models. that is the only reason why.
if that is a problem let me know. In a way I have to say that I prefer like this than the huge Config class.